### PR TITLE
Fix verification email

### PR DESCRIPTION
**Pull request by MikeOverkamp-diag**

### CODE REVIEW

**Comment by MikeOverkamp-diag** [comment]
I'm wondering if we want to allow the email field on the Verification model to blank? I feel it should not be blank, which is why I added the email in the fixtures. However, the field has `blank=True` set, which is why I also included the try.. except in the view function.

**Comment by jmsmkn** [comment]
I think I agree, AFAIK doesn't need to be blank any more, was left over from the initial implementation.

**Comment by MikeOverkamp-diag** [comment]
Removed the blank=True. It's probably still safest to keep the try.. except in place in case of any residual empty email fields.

File: app/grandchallenge/groups/views.py [hunk 1]
```diff
@@ -77,6 +77,10 @@ def get_result_label(self, result):
             is_verified = False
 
         if is_verified:
+            try:
```
Comment (jmsmkn) [inline app/grandchallenge/groups/views.py:80]: Are the changes in this file required now?

**Comment by jmsmkn** [comment]
> Removed the blank=True. It's probably still safest to keep the try.. except in place in case of any residual empty email fields.

I've checked and it's safe.

**Review by jmsmkn — APPROVED** [review]

**Pull request merged**